LayoutTests/inspector/timeline/timeline-event-performance-mark.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html>
+<head>
+<script src="../../http/tests/inspector/resources/inspector-test.js"></script>
+<script src="./resources/timeline-event-utilities.js"></script>
+<script>
+
+TestPage.registerInitializer(() => {
+    InspectorTest.TimelineEvent.captureTimelineWithMarker = function({expression, markerType, timelineType}) {
```

Comment:
You might as well put this in `timeline-event-utilities.js` to be shared with other tests if needed. I could see a test being written for the Load event marker.

Reply:
Sure, will do.